ui/components/app/nfts-items/nfts-items.js

```diff
@@ -169,9 +169,10 @@ export default function NftsItems({
         {isExpanded ? (
           <Box display={DISPLAY.FLEX} flexWrap={FLEX_WRAP.WRAP} gap={4}>
             {nfts.map((nft, i) => {
-              const { image, address, tokenId, name } = nft;
-              const nftImage = getAssetImageURL(image, ipfsGateway);
+              const { image, address, tokenId, name, imageOriginal } = nft;
+              const nftImage = getAssetImageURL(imageOriginal, ipfsGateway);
```

Comment:
Can you remind me why we use `imageOriginal` here? Is this a larger image?

Reply:
Yup, imageOriginal is the image URl on IPFS . Then we are using the `getAssetUrl ` method to get the value here
`const nftImage = getAssetImageURL(imageOriginal, ipfsGateway);`